package.json

```diff
@@ -65,6 +66,7 @@
     "gl-matrix": "3.3.0",
     "lodash": "4.17.21",
     "luxon": "^2.5.2",
+    "mapbox-gl": "^2.12.1",
```

Comment:
Why adding this dependency? We are using a fork of mapbox-gl. In other places we are adding an empty dependency if it's needed:
```
"mapbox-gl": "npm:empty-npm-package@1.0.0",
```

Reply:
because is a dependency of react-map-gl and to avoid a warning, I can remove it